app/views/shared/vacancy/_jobseeker_view.html.haml

```diff
@@ -5,7 +5,10 @@
 
 .jobs-banner
   .govuk-width-container
-    = render 'vacancies/back_link'
```

Comment:
This is the only place where the back_link partial is used. Please delete the partial and the partial's tests and translations 🗑️ 🔥